concat-kdf/Cargo.toml

```
@@ -0,0 +1,31 @@
+[package]
+name = "concat-kdf"
+version = "0.1.0"
+authors = ["Ruslan Piasetskyi <ruslan.piasetskyi@gmail.com>"]
+license = "MIT OR Apache-2.0"
+homepage = "https://github.com/RustCrypto/KDFs/"
+repository = "https://github.com/RustCrypto/KDFs/"
+description = "Concatenation Key Derivation Function (Concat KDF)"
+keywords = ["crypto", "concat-kdf", "KDF", "NIST"]
+categories = ["cryptography", "no-std"]
+readme = "README.md"
+edition = "2018"
```

```suggestion
edition = "2021"
rust-version = "1.58.1"
```

...although I'm unclear why the MSRV is 1.58.1. Are you actually using features introduced in that version? Otherwise a 2021 edition crate is MSRV 1.56.